docs/docs/reference/errors.md

```diff
@@ -1,6 +1,3 @@
 # Errors
 
-While you may not want to see them, informative errors help you design better workflows.
```

Comment:
Worth adding a module-level docstring here?

Reply:
maybe, what do you think it should say? to me it feels a bit self-explanatory